README.md

```diff
@@ -81,7 +81,7 @@ Generates Swift snippets from colors, fonts and layers.
 
   ```swift
   final class RadialGradientView: UIView {
-  
+
```

Comment:
Did you remove newline symbol from code snippet?

Reply:
Nope, it's just my text-editor was clean not-important empty symbols and also newline still here.